samples/central-login/.env.example

```diff
@@ -0,0 +1,7 @@
+CLIENT_ID=
```

Comment:
Is this just here in case someone wanted environment values specifically for the central-login app? Do we want placeholders or just leave them empty?

Reply:
We can totally add placeholders.

Reply:
I think placeholders in the .env example files are a useful hint for developers if they don't have the tutorial to hand, so would see them as worth having